pkg/machines/components/storagePools/storagePoolList.jsx

```diff
@@ -46,7 +46,7 @@ export class StoragePoolList extends React.Component {
                     <BreadcrumbItem to='#'>
                         {_("Virtual Machines")}
                     </BreadcrumbItem>
-                    <BreadcrumbItem isActive>
+                    <BreadcrumbItem to="#" isActive>
```

Comment:
This seems unrelated and also wrong. when I now click on `Storage Pools` it takes me to `Virtual Machines`. Before it was not clickable which made more sense.

Reply:
Oh sorry, that was not meant to be here.